config/compose.toml

```diff
@@ -37,6 +37,7 @@ name = "schema"
 
 [services.credential]
 name = "credential"
+service_endpoint = "http://localhost:8080"
```

Comment:
should this be in another section? ideally this is provided once for the entire server not for each service

Reply:
agree, I'll put it in top level config and propagate that down so every service has it available